airbyte-webapp/src/components/GroupControls/GroupControls.module.scss

```diff
@@ -6,7 +6,6 @@ $group-spacing: variables.$spacing-xl;
 $border-width: variables.$border-thick;
 
 .container {
-  margin-bottom: 27px;
```

Comment:
Removed this because we don't want this spacing in the Connector Builder. To not affect the look of the ConnectorForm, I opted to wrap the ArraySection and ConditionSection components in `SectionContainers` instead: https://github.com/airbytehq/airbyte/pull/20004/files#diff-359ca8d25eceab9f588e4ca17095865f532588fecd0d7d9f2fc76458c1b1f10bR66

This is similar to how we add the correct spacing in FormSection